packages/core/useMediaControls/index.ts

```diff
@@ -310,14 +302,32 @@ export function useMediaControls(target: MaybeRef<HTMLMediaElement | null | unde
     if (!el)
       return
 
-    el.loop = unref(options.loop)!
-    el.controls = unref(options.controls)!
-    el.muted = unref(options.muted)!
-    el.preload = unref(options.preload)!
-    el.autoplay = unref(options.autoplay)!
+    const loop = unref(options.loop)
+    if (loop) el.loop = loop
+
+    const controls = unref(options.controls)
+    if (controls) el.controls = controls
+
+    const muted = unref(options.muted)
+    if (muted) el.muted = muted
```

Comment:
Hmm, maybe it would be better to check for the existence of the property rather than the actual value. For example if the user set something like this, then the video would still be muted. I think the settings in `useMediaControls` should override any attributes.

```ts
useMediaControls(target, {
  muted: false
})
```
and the video element had
```html
<video muted></video>
```
The expected outcome of above should be `muted: false` but it would be `muted: true`.

But in this case below
```ts
useMediaControls(target)
```
and the video element had
```html
<video muted></video>
```
The expected outcome should be `muted: true`

Reply:
I Agree